src/services/tx/tx-sender/dispatch.ts

```diff
@@ -50,7 +50,10 @@ export const dispatchTxProposal = async ({
     proposedTx = await proposeTx(chainId, safeAddress, sender, safeTx, safeTxHash, origin)
   } catch (error) {
     if (txId) {
-      txDispatch(TxEvent.SIGNATURE_PROPOSE_FAILED, { txId, error: asError(error) })
+      txDispatch(TxEvent.SIGNATURE_PROPOSE_FAILED, {
+        txId,
+        error: asError(error),
+      })
```

Comment:
Unnecessary formatting
```suggestion
      txDispatch(TxEvent.SIGNATURE_PROPOSE_FAILED, { txId, error: asError(error) })
```